torch/_inductor/codecache.py

```diff
@@ -2652,22 +2656,27 @@ def _cutlass_include_paths() -> List[str]:
     ]
 
 
+def _lib_and_rpath_flags(path):
+    # -rpath ensures the DLL can find its dependencies when loaded, even if the
+    # library path is non-standard.
+    return [f"-L{path}", "-Xlinker", f"-rpath={path}"]
+
+
 def _cuda_lib_options() -> List[str]:
+    _set_gpu_runtime_env()  # cpp_extension consults the env
     from torch.utils import cpp_extension
 
+    lpaths = cpp_extension.library_paths(cuda=True) + [
+        sysconfig.get_config_var("LIBDIR")
+    ]
     extra_ldflags: List[str] = []
     if is_linux():
-        extra_lib_dir = "lib64"
```

Comment:
Seems we are not including this extra lib dir anymore for all linux versions. Would this affect non-fbcode cases?

Reply:
Actually yeah. I realize that `_transform_cuda_paths_for_fbcode` actually handles this case too, we just need to call it unconditionally (and probably rename it to drop the 'fbcode' part)